Readability.js

```diff
@@ -462,6 +477,37 @@ Readability.prototype = {
     return node && node.nextElementSibling;
   },
 
+  /**
+   * Like _getNextNode, but for DOM implementations with no
+   * firstElementChild/nextElementSibling functionality...
```

Comment:
Not sure if xmldom will work with a pure HTML-based DOM... and yes, the fix is only for jsdom; Firefox will cope (as will webkit, I believe).